lib/travis/build/addons/rethinkdb.rb

```diff
@@ -0,0 +1,44 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Rethinkdb < Base
+        SUPER_USER_SAFE = true
+
+        RETHINKDB_GPG_KEY = '0x3A8F2399'
+
+        def after_prepare
+          sh.fold 'rethinkdb' do
+            sh.if "ENV['TRAVIS_OS_NAME'] != 'linux'" do
```

Comment:
Keep in mind that the build script has to be compiled by the `travis-build` app, then sent to the VM that actually runs it.

`ENV['TRAVIS_OS_NAME']` is the value defined on the machine that compiles it, not the one defined on the VM that runs the build.

Also, `bash` will not allow this condition in the `if`.